R/trace_generation.R

```diff
@@ -579,11 +579,15 @@ geom2trace <- list(
       # Convert days into milliseconds
       x <- as.numeric(x) * 24 * 60 * 60 * 1000
     }
-    L <- list(x=x,
-              y=data$y,
+    # if there is more than one y-value for a particular combination of 
+    # x, PANEL, and group; then take the _max_ y.
+    data$x <- x
+    dat <- plyr::ddply(data, c("x", "PANEL", if("group"%in%names(data))"group"), 
+                       plyr::summarise, count = max(y))
+    L <- list(x=dat$x,
+              y=dat$count,
               type="bar",
               name=params$name,
-              text=data$text,
```

Comment:
I was being overly paranoid since it won't make sense unless `stat="identity"`, but it looks as though `data$text` whenever the stat is not the identity.  

Even when `stat="identity"`, a text aesthetic still might not make much sense (for example, the bar-category-names test), so in a026b9b, I added an extra check.

(BTW, this link should be more stable -- https://gallery.shinyapps.io/ggtree/ )